tutorials/understanding-gov/index.md

```diff
@@ -300,7 +300,25 @@ $ simd tx gov vote 1 yes --from alice --keyring-backend test
 $ simd tx gov vote 1 no --from bob --keyring-backend test
 ```
 
-<!-- TODO -->
+After waiting the voting period, you can see that the proposal passed.
+
+```sh
+$ simd query gov proposal 1 --output json | jq .status
+```
+
+This is because the governance proposal weight the votes given the amount of token staked. Alice had staked tokens, while Bob had no token staked. So Bob's vote was not took into consideration in the tally of the result.
```

Comment:
I'm not sure what you mean by this.

Reply:
* Alice votes Yes with 1 million ATOM.
* Alice unstakes her million ATOM and waits 3 weeks.
* Alice sends her million to Bob.
* Bob votes Yes with 1 million ATOM.

This should not be allowed. It was possible because the voting period was longer than the unsticking period.

Reply:
This cannot happen because the vote weight is calculated at vote tally. So only Bob's vote will be counted in that case (if he staked).

Reply:
Mmh, I don't like O(n) actions in blockchain 🤣 
Are you sure there is nothing like intermediate tally, and when unstaking, your vote is updated on all proposals?

Reply:
Lol